data/apply-planet_australia_suburb_recast.sql

```diff
@@ -0,0 +1,31 @@
+-- Suburbs in Australia are treated more like cities, so this recasts them to  place=town
+
+with bbox as
```

Comment:
Nit: This looks not like a box to me... maybe say `bounds` instead of `bbox`?

Here and elsewhere...